src/scripting.c

```diff
@@ -1282,14 +1282,14 @@ void scriptingInit(int setup) {
 
 /* Release resources related to Lua scripting.
  * This function is used in order to reset the scripting environment. */
-void scriptingRelease(void) {
-    dictRelease(server.lua_scripts);
+void scriptingRelease(int async) {
+    async ? freeLuaScriptsAsync(server.lua_scripts) : dictRelease(server.lua_scripts);
```

Comment:
i don't like this pattern.
ternary operators are commonly used to pick a value of of two options (when you need the value).
not when you wanna decide between two actions.
```suggestion
    if (async)
        freeLuaScriptsAsync(server.lua_scripts);
    else
        dictRelease(server.lua_scripts);
```

Reply:
After 6.2, I think we should come up with a rough outline of the style expectations. We'll never be able to format Redis, but these types of comments are very subjective